gwpy/cli/cliproduct.py

```diff
@@ -260,28 +260,57 @@ def arg_channels(cls, parser):
         """Add an `~argparse.ArgumentGroup` for channel options
         """
         group = parser.add_argument_group(
-            'Data options', 'What data to load')
-        group.add_argument('--chan', type=str, nargs='+', action='append',
-                           required=True, help='channels to load')
-        group.add_argument('--start', type=to_gps, nargs='+', action='append',
-                           help='Starting GPS times (required)')
-        group.add_argument('--duration', type=to_s, default=10,
-                           help='Duration (seconds) [10]')
+            'Data options',
+            'What data to load',
+        )
+        group.add_argument(
+            '--chan',
+            type=str,
+            nargs='+',
+            action='append',
+            required=True,
+            help='channels to load',
+        )
+        group.add_argument(
+            '--start',
+            type=to_gps,
+            nargs='+',
+            action='append',
+            help='Starting GPS times (required)',
+        )
+        group.add_argument(
+            '--duration',
+            type=to_s,
+            default=10,
+            help='Duration (seconds) [10]',
+        )
         return group
 
     @classmethod
     def arg_data(cls, parser):
         """Add an `~argparse.ArgumentGroup` for data options
         """
         group = parser.add_argument_group(
-            'Data source options', 'Where to get the data')
+            'Data source options',
+            'Where to get the data',
+        )
         meg = group.add_mutually_exclusive_group()
-        meg.add_argument('-c', '--framecache', type=os.path.abspath,
-                         help='read data from cache')
-        meg.add_argument('-n', '--nds2-server', metavar='HOSTNAME',
-                         help='name of nds2 server to use, default is to '
-                         'try all of them')
-        meg.add_argument('--frametype', help='GWF frametype to read from')
+        meg.add_argument(
+            '-c',
+            '--framecache',
+            type=os.path.abspath,
+            help='read data from cache',
```

Comment:
help="Path to file containing a list of paths to frame files containing data to plot"